compliance/virtualmachine/metrics/metrics.go

```diff
@@ -0,0 +1,57 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stackrox/rox/pkg/metrics"
+)
+
+// IndexReportsReceived is a counter for the number of virtual machine index reports received.
+var IndexReportsReceived = prometheus.NewCounter(
+	prometheus.CounterOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.ComplianceSubsystem.String(),
+		Name:      "virtual_machine_relay_index_reports_received_total",
+		Help:      "Total number of virtual machine index reports received by this Relay",
+	},
+)
+
+// IndexReportsNotRelayed is a counter for the number of virtual machine index reports that failed to get relayed to
+// sensor for various reasons.
+var IndexReportsNotRelayed = prometheus.NewCounter(
+	prometheus.CounterOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.ComplianceSubsystem.String(),
+		Name:      "virtual_machine_relay_index_reports_not_relayed_total",
+		Help:      "Total number of virtual machine index reports failed to get relayed to sensor by this Relay",
+	},
+)
+
+// IndexReportsRelayed is a counter for the number of virtual machine index reports relayed to sensor.
+var IndexReportsRelayed = prometheus.NewCounter(
```

Comment:
I would have probably combined this w/ `IndexReportsNotRelayed` into a single metric distinguished by a label. E.g. `virtual_machine_relay_index_reports_relayed_total{status="fail"}` and `virtual_machine_relay_index_reports_relayed_total{status="success"}`.